examples/metal-perfscale-cpt-data-path.yaml

```diff
@@ -0,0 +1,328 @@
+      workerNodesType.keyword: ""
+      workerNodesCount: 4
+      benchmark.keyword: k8s-netperf
+      ocpVersion: {{ version }}
+      networkType: OVNKubernetes
+      not:
+        stream: okd
+
+    metrics :
+    - name:  n2n-tput-64-1p
+      parallelism: 1
+      profile: TCP_STREAM
+      messageSize: 64
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: throughput
+      agg:
+        value: throughput
+        agg_type: avg
+
+    - name:  n2n-tput-64-2p
+      parallelism: 2
+      profile: TCP_STREAM
+      messageSize: 64
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: throughput
+      agg:
+        value: throughput
+        agg_type: avg
+
+    - name:  n2n-tput-1024-1p
+      parallelism: 1
+      profile: TCP_STREAM
+      messageSize: 1024
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: throughput
+      agg:
+        value: throughput
+        agg_type: avg
+
+
+    - name:  n2n-tput-1024-2p
+      parallelism: 2
+      profile: TCP_STREAM
+      messageSize: 1024
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: throughput
+      agg:
+        value: throughput
+        agg_type: avg
+
+    - name:  n2n-tput-4096-1p
+      parallelism: 1
+      profile: TCP_STREAM
+      messageSize: 4096
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: throughput
+      agg:
+        value: throughput
+        agg_type: avg
+
+    - name:  n2n-tput-4096-2p
+      parallelism: 2
+      profile: TCP_STREAM
+      messageSize: 4096
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: throughput
+      agg:
+        value: throughput
+        agg_type: avg
+
+    - name:  n2n-tput-8192-1p
+      parallelism: 1
+      profile: TCP_STREAM
+      messageSize: 8192
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: throughput
+      agg:
+        value: throughput
+        agg_type: avg
+
+    - name:  n2n-tput-8192-2p
+      parallelism: 2
+      profile: TCP_STREAM
+      messageSize: 8192
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: throughput
+      agg:
+        value: throughput
+        agg_type: avg
+
+    - name:  n2n-ltcy-1024-1p
+      parallelism: 1
+      profile: TCP_RR
+      messageSize: 1024
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: latency
+      agg:
+        value: latency
+        agg_type: avg
+
+    - name:  n2n-ltcy-1024-2p
+      parallelism: 2
+      profile: TCP_RR
+      messageSize: 1024
+      driver.keyword: netperf
+      hostNetwork: true
+      acrossAZ: true
+      metric_of_interest: latency
```

Comment:
Since we are looking at latency,  we change `direction` to be if it goes positive (increase)?